gemviz/mainwindow.py

```diff
@@ -3,7 +3,11 @@
 import __init__
 import utils
 from app_settings import settings
+from PyQt5.QtCore import QUrl
 
+LOCALHOST_URL = "http://localhost:8000"
+TESTING_URL="http://otz.xray.aps.anl.gov:8000"
```

Comment:
While it may be useful for development, we should _bake_ this into the production version.  Needs a TODO comment here to be removed for production.

Reply:
Done